resolwe_bio/tools/create_track_hubs.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Create UCSC track hub files."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+
+parser = argparse.ArgumentParser(description="Create UCSC track hub files")
+parser.add_argument('sample_names', help="Sample names")
```

Comment:
Refactor this to a required positional argument ('--name') with multiple values (`nargs='+'`).